draft-js-mention-plugin/src/MentionSuggestions/index.js

```diff
@@ -58,7 +58,9 @@ export default class MentionSuggestions extends Component {
           focusedOptionIndex: size - 1,
         });
       }
-
+      if (!this.props.store.getAllSearches().has(this.activeOffsetKey)) {
```

Comment:
ok, so then the plugin doesn't throw an error, but won't work anyway. is this correct?